File: 3396-valid-word/3396-valid-word.java
import java.util.regex.*;
class Solution {
    public boolean isValid(String word) {
        return (word.length()>=3 && word.matches("[A-Za-z0-9]+") && Pattern.compile("[aeiouAEIOU]").matcher(word).find() && Pattern.compile("[b-df-hj-np-tv-zB-DF-HJ-NP-TV-Z]").matcher(word).find());
    }
}

File: 3396-valid-word/3396-valid-word.py
class Solution:
    def isValid(self, word: str) -> bool:
        return True if len(word)>=3 and word.isalnum() and re.search(r'[aeiouAEIOU]',word) and re.search(r'[b-df-hj-np-tv-zB-DF-HJ-NP-TV-Z]',word) else False